docs/README.md

```diff
@@ -1,15 +1,29 @@
+[La version française suit.](#---------------------------------------------------------------------)
+
 # Tracker documentation
 
 Welcome to the [`tracker`](https://github.com/cds-snc/tracker) documentation!  
 Here you will find the following documents to help gain an understanding around the system.
 
-| document | description | 
-| -------- | ----------- | 
-| [additions.md](additions.md) | This document outlines the location of key functionality to facilitate making additions or changes to the system. |
-| [deploy.md](deploy.md) | This document outlines a basic deployment workflow. |
-| [data.md](data.md) | This documents outlines the input and output data schema with descriptions of fields and their possible values. |
-| [lambda-deploy.md](lambda-deploy.md) | This document outlines the process of deploying and using AWS lambda scanners. |
-| [local-instructions.md](local-instructions.md) | This document contains step-by-step instructions on how to setup a development verison of the project locally. |
-| [ownership.md](ownership.md) | This document outlines the algorithm used to determine what organization owns what domain. |
+| document | description |
```

Comment:
Should these table headers still be in English?

Reply:
those are bilingual words :D

Reply:
I see, cool

Also I commented on the wrong line, should have been L23